experimental/dds/tree2/src/test/editMinter.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { DefaultChangeFamily, DefaultChangeset, DefaultEditBuilder } from "../feature-libraries";
+
+export type Editor = (builder: DefaultEditBuilder) => void;
+
+let builtChangeset: DefaultChangeset | undefined;
+
+export function makeEditMinter(
+	family: DefaultChangeFamily,
+	editor: Editor,
+): () => DefaultChangeset {
+	const innerEditor = family.buildEditor((change) => {
+		assert(builtChangeset === undefined);
```

Comment:
is it intentional that `builtChangeset` is scoped to the module and not inside `makeEditMinter`? Seems weird to me that creating multiple edit minters and interleaving their calls shares this state.

Reply:
It didn't occur to me to scope it in but that's a good idea. :)